src/shogun/clustering/unittest_random_initialization.cc

```diff
@@ -0,0 +1,52 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
```

Comment:
Can you use BSD?
See [here](https://gist.github.com/karlnapf/a535d810b7d78deaee81)
Put your name. Check other files for how to do it